scss/atoms/typography/_typography.scss

```diff
@@ -54,7 +54,7 @@ h6 {
 	font-size: em( 16px );
 	font-weight: 600;
 	line-height: 1;
-	text-transform: uppercase;
+	text-transform: uppercase; // FIXME Is this opinionated?
 }
```

Comment:
That should probably be moved to a variable - so they have the ability to set that.